tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+
+		switch ( $page ) {
+			case 'add-event':
+				if (
+					'add' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			case 'edit-event':
+				if (
+					'edit' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			default:
+				$is_page = false;
+				break;
+		}
+
+		return $is_page;
+	}
+
+	/**
+	 * Do the output
+	 *
+	 * @return string
+	 */
+	public function display() {
+		$symbol_list = $this->symbol_list_select_options();
+
+		// if no symbols are entered in Settings, do not display Symbol or Position on CE form
+		if ( empty( $symbol_list ) ) {
+			return '';
+		}
+
+		?>
+
+		<table class="tribe-section-content">
+			<colgroup>
+				<col class="tribe-colgroup tribe-colgroup-label">
+				<col class="tribe-colgroup tribe-colgroup-field">
+			</colgroup>
+
+			<tr class="tribe-section-content-row">
+				<td class="tribe-section-content-label">
+					<?php tribe_community_events_field_label( 'EventCurrencySymbol', __( 'Currency Symbol:', 'tribe-extension' ) ); ?>
+				</td>
+				<td class="tribe-section-content-field">
+					<select
+						id="EventCurrencySymbol"
+						aria-label="<?php esc_html_e( 'Currency Symbol', 'tribe-extension' ); ?>"
+						name="EventCurrencySymbol"
+						class="event-cost-currency-symbol tribe-dropdown"
+						placeholder="<?php esc_attr_e( 'Select a currency symbol', 'tribe-extension' ); ?>"
+					>
+						<?php echo $symbol_list; ?>
+					</select>
+				</td>
+			</tr>
+
+		</table>
+
+		<?php
+		// allow disabling the Before/After choice on the CE form
+		$disabled = tribe_get_option( 'tribe_ext_ce_cost_currency_symbol_position_disabled' );
+
+		if ( empty( $disabled ) ) {
+			?>
+			<table class="tribe-section-content">
+				<colgroup>
+					<col class="tribe-colgroup tribe-colgroup-label">
+					<col class="tribe-colgroup tribe-colgroup-field">
+				</colgroup>
+
+				<tr class="tribe-section-content-row">
+					<td class="tribe-section-content-label">
+						<?php tribe_community_events_field_label( 'EventCurrencyPosition', __( 'Symbol Position:', 'tribe-extension' ) ); ?>
+					</td>
+					<td class="tribe-section-content-field">
+						<select
+							id="EventCurrencyPosition"
+							aria-label="<?php esc_html_e( 'Symbol Position', 'tribe-extension' ); ?>"
+							name="EventCurrencyPosition"
+							class="event-cost-currency-symbol-position tribe-dropdown"
+							placeholder="<?php esc_attr_e( 'Override the symbol position', 'tribe-extension' ); ?>"
+						>
+							<?php echo $this->symbol_position_list_select_options(); ?>
+						</select>
+						<p>
+							<?php _e( 'The position of the currency symbol in the cost string.', 'tribe-extension' ); ?>
+							<br>
+							<?php printf( __( 'Leave blank to use the default: %s', 'tribe-extension' ), $this->get_symbol_position_phrase( $this->get_symbol_position_default() ) ); ?>
```

Comment:
would encourage `esc_html__()` unless there's a specific reason that would complicate things here (like messing with output display of the various symbols/settings/etc.)